arches/install/arches-app-templates/setup.py

```diff
@@ -10,6 +10,7 @@
     license="GNU AGPL3",
     packages=find_packages(),
     include_package_data=True,
+    python_requires=">=3.8",
```

Comment:
My thought was that in 7.6 (LTS) we can start requiring Python 3.11 (min), and possibly support 3.12 also if Django 4.2 will support it.